src/client/components/additionalConfigs/settings/settings.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+import './settings.css';
+
+const settingsIcon = <FontAwesomeIcon icon={faCog} />;
```

Comment:
I would suggest to create a file with all icons for your project and import it from there